test/alda/lisp/pitch_test.clj

```diff
@@ -63,4 +106,28 @@
                         (key-signature "f+")))
           piano     (get-instrument s "piano")
           f-sharp-4 (calculate-pitch :f [] 4 (:key-signature piano))]
-      (is (= f-sharp-4 (calculate-pitch :f [:sharp] 4 {}))))))
+      (is (= f-sharp-4 (calculate-pitch :f [:sharp] 4 {})))))
+
+  (testing "ionian should be the same as major"
+    (let [s     (score
+                 (part "piano"
+                   (key-sig [:c :ionian])))
+          t     (score
+                  (part "piano"
+                    (key-sig [:c :major])))
+         piano  (get-instrument s "piano")
+         piano2 (get-instrument t "piano")]
+     (is (= (:key-signature piano2)
+            (:key-signature piano)))))
```

Comment:
Right now, because of the bug noted above, I think this test is only passing because `(= nil nil)` is true. I think we should also check for the expected value, like this:

```clojure
(is (= (:key-signature piano2)
       (:key-signature piano)
       {}))
```

Reply:
I still think we should make this change, to guard against testing the equality of 2 things that are both wrong.